{"name": "test_abc", "status": "broken", "statusDetails": {"message": "NameError: name 'id1' is not defined", "trace": "test_readReasposedata = [7621578]\n\n    def test_abc(test_readReasposedata):\n        print(\"Hello\")\n>       print(id1)\nE       NameError: name 'id1' is not defined\n\nStudentDetails\\test_AddStudentDetail.py:43: NameError"}, "start": 1688481262917, "stop": 1688481262918, "uuid": "5c025fa5-7f06-4e0c-b70b-9de08e065fa7", "historyId": "10ba7b1609ae55841158618663860783", "testCaseId": "10ba7b1609ae55841158618663860783", "fullName": "StudentDetails.test_AddStudentDetail#test_abc", "labels": [{"name": "parentSuite", "value": "StudentDetails"}, {"name": "suite", "value": "test_AddStudentDetail"}, {"name": "host", "value": "UT-LTP-949"}, {"name": "thread", "value": "9988-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "StudentDetails.test_AddStudentDetail"}]}